robotcontainer.py

```diff
@@ -203,5 +205,18 @@ def configureButtonBindings(self):
             ClimberTucked(self.climber)
         )
 
+        # manual elevator
+        NetworkTableButton(constants.kElevatorManualModeKey).whileTrue(
+            ElevatorManualMode(self.elevator).repeatedly()
+        )
+
+        ModifiableJoystickButton(self.operatorInterface.elevatorManualUp).whileTrue(
```

Comment:
wouldn't this just cancel the other elevatormanualmode command when its activated?

Reply:
alternatively what you can do is just have it that when you press either manual up or down, it sets the elevator into manual mode, which can be overwritten with any of the other state setters, basically have a ParallelCommandGroup of the both, or have elevatormanualup and elevatormanualdown set the state to manual

Reply:
> alternatively what you can do is just have it that when you press either manual up or down, it sets the elevator into manual mode, which can be overwritten with any of the other state setters, basically have a ParallelCommandGroup of the both, or have elevatormanualup and elevatormanualdown set the state to manual

how would I do this without having the default command override the state

Reply:
> how would I do this without having the default command override the state
default command needs to differ from standard setting commands in that it will only set the state when no longer in manual

would also mean you need a "release" button of sorts for manual mode

I anticipate using stuff like this for fine on field adjustments as needed
the other option is someone grinds out making a real control panel with switches for manual mode but thats a lot of work

Reply:
I'm giving my 2 cents but having something that affects elevator not actually requiring the elevator is very sketchy and contradicts part of the purpose of command based, but also at the end of the day its about if it works or not

Reply:
it do be working and the "requirement" is manual mode inside the command

Reply:
marking convo as resolved with the idea that I don't care too hard but will be criticizing

Reply:
might make the buttons themselves switch to manual mode and have a separate button to exit manual mode